indexer/services/roundtable/src/config.ts

```diff
@@ -64,6 +64,10 @@ export const configSchema = {
     // 1 hour
     default: 3_600_000,
   }),
+  LOOPS_INTERVAL_MS_UPDATE_COMPLIANCE_DATA: parseInteger({
+    // 5 minutes
+    default: 300_000,
```

Comment:
nit: maybe not for this PR, but at some point we should just create a bunch for constants like `5 * FIVE_MINUTES_IN_MS` which would be cleaner